activation/handler.go

```diff
@@ -252,6 +263,11 @@ func (h *Handler) validateNonInitialAtx(ctx context.Context, atx *types.Activati
 		return fmt.Errorf("prevATX declared, but initial Post is included")
 	}
 
+	if prevAtx.NumUnits < atx.NumUnits {
+		atx.SetEffectiveNumUnits(prevAtx.NumUnits)
+	} else {
+		atx.SetEffectiveNumUnits(atx.NumUnits)
+	}
```

Comment:
issue #3963 stated
> The changed numUnits value becomes effective in the TargetEpoch of the ATX, when numUnits was lowered, but when numUnits is increased it becomes effective one epoch after TargetEpoch.
A changed VRFNonce always becomes effective in TargetEpoch.

how is this achieved?